src/sass/modules/_m-modal.scss

```diff
@@ -152,7 +152,7 @@
   white-space: nowrap;
 }
 
-.va-modal-close {
+button.va-modal-close {
```

Comment:
The close button's size was being overridden by another style because the `<LoginModal>` was a child of a `.form-progress` element (`FormPage`?).